bindata/network/ovn-kubernetes/common/001-crd.yaml

```diff
@@ -3283,6 +3283,50 @@ spec:
               layer2:
                 description: Layer2 is the Layer2 topology configuration.
                 properties:
+{{- if .OVN_PRE_CONF_UDN_ADDR_ENABLE }}
+                  defaultGatewayIPs:
```

Comment:
this looks exact same to me from https://github.com/ovn-kubernetes/ovn-kubernetes/blob/7c047281d46a6e5f154a0f5d3feb8866af3619f4/dist/templates/k8s.ovn.org_clusteruserdefinednetworks.yaml.j2#L97